tango_simlib/utilities/fandango_json_parser.py

```diff
@@ -78,7 +78,6 @@ def preprocess_attribute_types(self, attribute_data):
         """Convert the attribute data types from strings to the TANGO types.
         """
         for attr, attr_config in attribute_data.items():
-            
             # assign 'READ_WRITE' to all attributes with 'WT_UNKNOWN'
             if attr_config['writable'] not in ['READ', 'WRITE', 'READ_WRITE']:
```

Comment:
I had a look through the pytango code and tango docs and I think there is one extra option we should include here, which is also valid:  `READ_WITH_WRITE`. 
See these links:
http://tango-controls.readthedocs.io/en/latest/development/advanced/reference.html#the-attribute-read-write-type
https://github.com/tango-controls/pytango/blob/71236c9/tango/base_types.py#L565-L572
https://github.com/tango-controls/pytango/blob/a4dc9c5/ext/enums.cpp#L173-L178

Reply:
Thanks for pointing this out.